pyproject.toml

```diff
@@ -75,6 +75,15 @@ test = [
   "opencv-python",
 ]
 
+notebooks = [
```

Comment:
Would `test_notebooks` be a better name?

Reply:
I am indifferent to a change here. Technically these are also the dependencies needed to execute notebooks. Your call how we do name this.

Reply:
I renamed to `test_notebooks`